chips/cc26x0/src/udma.rs

```diff
@@ -0,0 +1,56 @@
+///
```

Comment:
Awesome that you started on this!

Reply:
Thanks! I'm going to shoot for an MVP right now, but I think there's a larger conversation about what the DMA manager should look like. It seems like the SAM4L model might be the closest analog...